python/delta/tables.py

```diff
@@ -67,17 +81,26 @@ def delete(self, condition=None):
         else:
             self._jdt.delete(self._condition_to_jcolumn(condition))
 
+    @since(0.4)
     def update(self, condition=None, set=None):
         """
         Update data from the table on the rows that match the given ``condition``,
         which performs the rules defined by ``set``.
 
+        Example::
+
+            # condition using SQL formatted string
+            deltaTable.update("eventType = 'clck'", { "eventType": "'click'" } )
+
+            # condition using Spark SQL functions
+            deltaTable.update(col("eventType") == "clck", { "eventType": lit("click") } )
```

Comment:
Maybe use named parameters for these examples to make it clearer?

Reply:
what is unclear about using positional arguments here?

Reply:
It's not unclear, but I found the merge example with the named args a little easier to read